dependencies.json

```diff
@@ -1,8 +1,10 @@
 {
   "*": {
     ">=3118": [
```

Comment:
I believe this needs to be `>=3114` as per [this comment](https://github.com/equinusocio/material-theme/issues/925#issuecomment-236635324)